source/includes/administration/_feedback_settings.md

```diff
@@ -0,0 +1,207 @@
+## Feedback Settings
+
+<!-------------------- FIND FEEDBACK SETTINGS -------------------->
+
+### Find feedback settings associated to an organization
+
+`GET /feedback/find?organizationId=:id`
+
+Retrieve the feedback settings associated to an organization.
+
+```shell
+# Retrieve the feedback settings
+curl "https://cloudmc_endpoint/api/v1/feedback/find?organizationId=fcda8d5a-0276-4de3-908f-b3bd0aff2491" \
```

Comment:
Could you add a note that if the organizationId query param is left blank, we will use the authenticated users organization.

Reply:
@jasondias9 @m-orfanos I added the precision in the docs.